packages/hash/frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/definition-tab/property-list-card/shared/property-type-form/property-type-schema.ts

```diff
@@ -67,7 +67,10 @@ export const getArrayItems = (
             oneOf: getArrayItems(data.itemIds, flattenedExpectedValues),
           },
           minItems: data.minItems,
-          maxItems: data.maxItems,
+          ...(data.minItems ? { minItems: data.minItems } : {}),
+          ...(data.maxItems && !data.infinity
+            ? { maxItems: data.maxItems }
+            : {}),
```

Comment:
I guess we don't need the `minItems: data.minItems` line anymore

Reply:
Good catch, removed redundant line in 0625667275599cc9f2841c62f9ff8fb7b67d01e2